library/init/meta/interactive.lean

```diff
@@ -709,24 +723,40 @@ hypotheses to `x, y, z`. The renaming functionality would be available only if
 the goal has a case tag.
 -/
 meta def case (ns : parse ident_*) (ids : parse $ (tk ":" *> ident_*)?) (tac : itactic) : tactic unit :=
```

Comment:
I guess I forgot to transfer this from my working copy. My best guess is that I meant
```suggestion
meta def case (ns : parse ident_*) (args : case_arg_parser) (tac : itactic) : tactic unit :=
```